web/client/components/resources/modals/Save.jsx

```diff
@@ -18,6 +18,30 @@ const MainForm = require('./fragments/MainForm');
 const ruleEditor = require('./enhancers/ruleEditor');
 const PermissionEditor = ruleEditor(require('./fragments/PermissionEditor'));
 
+/**
+ * Defines if the resource permissions are available or not.
+ * Actually GeoStore allow editing of the permission to the owner or to the administrators.
+ * For new resources the owner (will be) the current user so if owner is missing (as for new resources),
+ * this returns true.
+ * @param {object} user the user object, with role and name properties
+ * @param {object} resource the resource object, with attributes (owner)
+ */
+const canEditResourcePermission = (user = {}, resource) => {
+    // anonymous users can not edit
+    if (!user) {
+        return false;
+    }
+    const { role, name } = user;
+    if (role === 'ADMIN') {
```

Comment:
does this break the following ?
> Maps have owner, so only owner can edit permissions, the other users can only edit the resource

Reply:
Sorry, in the PR i forgot to tell that admin user can always edit everything